README.md

```diff
@@ -176,6 +190,32 @@ As a result you will get a `TransactionDetails` object with the following info i
     public final PurchaseInfo purchaseInfo;
 ```
 
+Protection against fake "markets"
+--------------------------------
+There are number of attacks which exploits some vulnerabilities of Google's Play Market.
+Among them is so-called *Freedom attack*: *Freedom* is special Android application, which
+intercepts application calls to Play Market services and substitutes them with fake ones. So in the
+  end attacked application *thinks* that it receives valid responses from Play Market.
+
+In order to protect from this kind of attack you should specify your `merchantId`, which
+can be found in your [Payments Merchant Account](https://payments.google.com/merchant).
+Selecting *Settings->Public Profile* you will find your unique `merchantId`
+
+**WARNING:** keep your `merchantId` in safe place!
```

Comment:
What do you mean with keep your merchant ID safe? can somebody do damage to me if he know what my merchant ID is?